src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -127801,6 +127801,4340 @@ static T F2<T>()
+                Diagnostic(ErrorCode.ERR_OverrideNotExpected, "F1").WithArguments("B1.F1<T>(T?)").WithLocation(14, 28),
+                // (15,31): error CS8822: Method 'B1.F2<T>(T?)' specifies a 'default' constraint for type parameter 'T', but corresponding type parameter 'T' of overridden or explicitly implemented method 'A.F2<T>(T?)' is constrained to a reference type or a value type.
+                //     internal override void F2<T>(T? t) where T : default { }
+                Diagnostic(ErrorCode.ERR_OverrideDefaultConstraintNotSatisfied, "T").WithArguments("B1.F2<T>(T?)", "T", "T", "A.F2<T>(T?)").WithLocation(15, 31),
+                // (19,58): error CS0449: The 'class', 'struct', 'unmanaged', 'notnull', and 'default' constraints cannot be combined or duplicated, and must be specified first in the constraints list.
+                //     internal override void F1<T>(T? t) where T : struct, default { }
+                Diagnostic(ErrorCode.ERR_TypeConstraintsMustBeUniqueAndFirst, "default").WithLocation(19, 58),
+                // (20,57): error CS0449: The 'class', 'struct', 'unmanaged', 'notnull', and 'default' constraints cannot be combined or duplicated, and must be specified first in the constraints list.
+                //     internal override void F2<T>(T? t) where T : class, default { }
+                Diagnostic(ErrorCode.ERR_TypeConstraintsMustBeUniqueAndFirst, "default").WithLocation(20, 57),
+                // (22,7): error CS0534: 'B3' does not implement inherited abstract member 'A.F1<T>(T?)'
+                // class B3 : A
+                Diagnostic(ErrorCode.ERR_UnimplementedAbstractMethod, "B3").WithArguments("B3", "A.F1<T>(T?)").WithLocation(22, 7),
+                // (24,28): error CS0115: 'B3.F1<T>(T?)': no suitable method found to override
+                //     internal override void F1<T>(T? t) where T : default, struct { }
+                Diagnostic(ErrorCode.ERR_OverrideNotExpected, "F1").WithArguments("B3.F1<T>(T?)").WithLocation(24, 28),
+                // (24,59): error CS0449: The 'class', 'struct', 'unmanaged', 'notnull', and 'default' constraints cannot be combined or duplicated, and must be specified first in the constraints list.
+                //     internal override void F1<T>(T? t) where T : default, struct { }
+                Diagnostic(ErrorCode.ERR_TypeConstraintsMustBeUniqueAndFirst, "struct").WithLocation(24, 59),
+                // (25,59): error CS0449: The 'class', 'struct', 'unmanaged', 'notnull', and 'default' constraints cannot be combined or duplicated, and must be specified first in the constraints list.
+                //     internal override void F2<T>(T? t) where T : default, class { }
+                Diagnostic(ErrorCode.ERR_TypeConstraintsMustBeUniqueAndFirst, "class").WithLocation(25, 59));
+        }
+
+        [Theory]
+        [CombinatorialData]
+        public void UnconstrainedTypeParameter_05(bool useCompilationReference)
```

Comment:
Think it would be good to have the sample which has the correct overrides at the end.